README.md

```diff
@@ -5,7 +5,8 @@
 ## Setup
 
 1. Run `npm install` to download all the required node modules
-2. TODO
+1. To build the front end `npx @angular/cli build --prod`
```

Comment:
Where do I run this @sampathsatti ?

When I try in `ODetect-Backend-Local`, I get this message:
```Cannot open assembly 'build': No such file or directory.```

I tried adding a `build` directory and running it again in `ODetect-Backend-Local` and get:
```Cannot open assembly 'build': File does not contain a valid CIL image."

Then I clone the front end and tried running it in `ODetect-Frontend-Local` and, again, I get the same thing.

Reply:
It would also be nice to have some reassurance that you use the `--prod` for both prod and dev deployments. (That's rihgt, isn't it?)